core/dbt/events/types.py

```diff
@@ -30,11 +30,12 @@
 # | W    | Node testing        |
 # | Y    | Post processing     |
 # | Z    | Misc                |
+# | T    | Test only           |
 #
 # The basic idea is that event codes roughly translate to the natural order of running a dbt task
 
 # can't use ABCs with @dataclass because of https://github.com/python/mypy/issues/5374
-@dataclass
+@dataclass  # type: ignore
```

Comment:
oh wow I didn't even think of doing this. we can get rid of the comment that says we can't do this then.